packages/zosjobs/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Change Log
 
 All notable changes to the Zowe z/OS jobs SDK package will be documented in this file.
-
+## Recent Changes
+- BugFix: "Changed default value for modify-jobs option in the zowe jobs command group to 2.0. This change results in calls to z/OSMF becoming synchronous, and a successful response from the modify, cancel, and delete commands indicates the requested action was completed successfully. [#1459](https://github.com/zowe/zowe-cli/issues/1459)
```

Comment:
Instead of "zowe jobs command group", should we say "CancelJobs and DeleteJobs classes" to be more SDK specific?

Also, unnecessary quotation mark (same as above)